clippy_lints/src/loops/needless_range_loop.rs

```diff
@@ -149,7 +149,7 @@ pub(super) fn check<'tcx>(
                         |diag| {
                             multispan_sugg(
                                 diag,
-                                "consider using an iterator",
+                                "consider using an iterator and enumerate",
```

Comment:
```suggestion
                                "consider using an iterator and enumerate()",
```

Reply:
Pushed commit with bless for this change.